lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -75,12 +75,14 @@ def fetch_logs
       return {} unless exists? && @containers.present?
 
       @containers.each_with_object({}) do |container_name, container_logs|
-        out, _err, _st = kubectl.run(
+        cmd = [
           "logs",
           @name,
           "--container=#{container_name}",
-          "--since-time=#{@deploy_started.to_datetime.rfc3339}"
-        )
+          "--since-time=#{@deploy_started.to_datetime.rfc3339}",
+        ]
+        cmd << "--tail=50" unless unmanaged?
```

Comment:
The assumption is that unmanaged pods (i.e. where you have a pod template in your deploy directory) are performing a deploy-time operation and you want to see all their output inline. When we dump logs from a managed pod however (e.g. on deployment failure), the line count could be absurd, making the summary section unusable.

In researching deploy failures that happened since I made the output changes, I found a couple examples of super long log dumps:
- [this one](https://shipit.shopify.io/shopify/plus-account-centre/production/deploys/451844) has 128 lines of stack trace
- [this one](https://shipit.shopify.io/shopify/facebook-commerce/staging/deploys/451871) has 126 lines of output that looks completely irrelevant to the deploy failure

Arguably, the stack trace case won't be helpful truncated to the last 50 lines. But then again, maybe the takeaway in such cases should be "I need to check Bugsnag"? I'm really not sure what the limit should be, but we should have _something_, even if it is really high. What do you think?